embedding-store-filter-parsers/langchain4j-embedding-store-filter-parser-sql/src/test/java/dev/langchain4j/store/embedding/filter/parser/sql/SqlFilterParserTest.java

```diff
@@ -0,0 +1,1693 @@
+                ))
+                .add(of(
+                        "age = 18",
+                        key("age").eq(18L)
+                ))
+                .add(of(
+                        "weight = 67.8",
+                        key("weight").eq(67.8d)
+                ))
+
+
+                // ne
+                .add(of(
+                        "name != 'Klaus'",
+                        key("name").ne("Klaus")
+                ))
+                .add(of(
+                        "age != 18",
+                        key("age").ne(18L)
+                ))
+                .add(of(
+                        "weight != 67.8",
+                        key("weight").ne(67.8d)
+                ))
+
+
+                // gt
+                .add(of(
+                        "name > 'Klaus'",
+                        key("name").gt("Klaus")
+                ))
+                .add(of(
+                        "age > 18",
+                        key("age").gt(18L)
+                ))
+                .add(of(
+                        "weight > 67.8",
+                        key("weight").gt(67.8d)
+                ))
+
+
+                // gte
+                .add(of(
+                        "name >= 'Klaus'",
+                        key("name").gte("Klaus")
+                ))
+                .add(of(
+                        "age >= 18",
+                        key("age").gte(18L)
+                ))
+                .add(of(
+                        "weight >= 67.8",
+                        key("weight").gte(67.8d)
+                ))
+
+
+                // lt
+                .add(of(
+                        "name < 'Klaus'",
+                        key("name").lt("Klaus")
+                ))
+                .add(of(
+                        "age < 18",
+                        key("age").lt(18L)
+                ))
+                .add(of(
+                        "weight < 67.8",
+                        key("weight").lt(67.8d)
+                ))
+
+
+                // lte
+                .add(of(
+                        "name <= 'Klaus'",
+                        key("name").lte("Klaus")
+                ))
+                .add(of(
+                        "age <= 18",
+                        key("age").lte(18L)
+                ))
+                .add(of(
+                        "weight <= 67.8",
+                        key("weight").lte(67.8d)
+                ))
+
+
+                // in
+                .add(of(
+                        "name IN ('Klaus', 'Francine')",
+                        key("name").in("Klaus", "Francine")
+                ))
+                .add(of(
+                        "age IN (18, 42)",
+                        key("age").in(18L, 42L)
+                ))
+                .add(of(
+                        "weight IN (67.8, 78.9)",
+                        key("weight").in(67.8d, 78.9d)
+                ))
+
+
+                // nin
```

Comment:
At first I wondered, _"what is this `nin` operator"_ but immediately understood it was `not in`.
I guess you want to keep it short, but `notIn` might be more readable?

Reply:
New naming:

```
isEqualTo
isNotEqualTo
isGreaterThan
isGreaterThanOrEqualTo
isLessThan
isLessThanOrEqualTo
isBetween
isIn
isNotIn
```

Took it from AssertJ. WDYT?